### DISCUSSION TITLE
Apollo Save Tool v1.1.0

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps4#50] [Announcements]
### Added

* New firmware support: 5.07, 7.51
* Improved save-game listing (game name, saves count)
* Save-game selection (press `Touchpad` to select)
  * Copy/Resign selected saves (Bulk Management)
* Detect if a save couldn't be unmounted

### Fixed

* Fixed Trophy-set listing bug
* Avoid issue when removing mount patches on exit

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps4/releases/tag/v1.1.0'>Apollo Save Tool v1.1.0</a>.</em>

**Comment by yc01121** [comment]
[CUSA05682.zip](https://github.com/bucanero/apollo-ps4/files/8536497/CUSA05682.zip)
HI 
There's something wrong with Horizon Zero Dawn: Complete Edition Data CUSA10218
Complete Edition can not use Horizon Zero Dawn standard data CUSA05682 (CE-3316-4)

PS: Thank you bring apollo to PlayStation Player. That 's Cool

**Comment by hidane07** [comment]
Hello
I just wanna tell you the apollo save not working for the game MGSV i download save game and i do juste what do explain but not working
please can you try to do it for me please i beg you

**Comment by dinogg13** [comment]
Hello Bucareno,
When will it be possible to do as save wizard (import a decrypted save that was modified for example gta v editor) and after then crypt this save in usb slot ?

Thanks

**Comment by ArabPixel** [comment]
Great tool, next step 10.50?
